src/Adapter/Manufacturer/ManufacturerProductSearchProvider.php

```diff
@@ -54,14 +54,14 @@ public function __construct(
      * @param ProductSearchQuery $query
      * @param string $type
      *
-     * @return array|bool
+     * @return array|int
```

Comment:
Should not be changed, the return type change, it's a BC

Reply:
It's not a BC : it's a private function.